xml/containers_basics.xml

```diff
@@ -4,83 +4,75 @@
   <!ENTITY % entities SYSTEM "entity-decl.ent">
     %entities;
 ]>
-
 <chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha-containers-basics">
  <title>Introduction to Linux Containers</title>
  <info>
-      <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
-        <dm:bugtracker>
-        </dm:bugtracker>
-      </dm:docmanager>
-    </info>
-    <para>
-  <emphasis>Linux containers</emphasis> are a lightweight virtualization
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+  </dm:docmanager>
+ </info>
+ <para>
+  <emphasis>Linux containers</emphasis> offer a lightweight virtualization
   method to run multiple virtual units (<quote>containers</quote>)
```

Comment:
(Yes I know I am picking on an unchanged line.)

I do not think that the word "units" is a good one. They are sometimes called "virtual environments" which I would prefer. What do others think?

Reply:
Agree. Virtual environments is better.